sdk/spring/azure-spring-boot-starter-active-directory/README.md

```diff
@@ -19,9 +19,9 @@ With Spring Starter for Azure Active Directory, now you can get started quickly
 [//]: # "{x-version-update-start;com.microsoft.azure:azure-active-directory-spring-boot-starter;current}"
```

Comment:
This line should also update to com.azure:azure-spring-boot-starter-active-directory, same as other README.md files